pkg/framework/spi/jwe/api.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jwe
+
+import "github.com/google/tink/go/keyset"
```

Comment:
I assume you are removing these imports from the rest of the SPI interfaces?

Reply:
they're now all removed.